### PR TITLE
Remove tus from Resumable Upload draft

### PR DESCRIPTION
**Pull request by Acconut**
Closes https://github.com/httpwg/http-extensions/issues/2349

### CODE REVIEW

**Comment by mnot** [comment]
I suspect a title like "HTTP Resumable Uploads" or "Resumable Uploads for HTTP" would be more clear -- not referring to HTTP will pull in readers looking for other things, and calling it a protocol reinforces that it's separate, when we're trying to make it an extension to an _existing_ protocol.

**Comment by Acconut** [comment]
Good idea to include HTTP in the title! I updated the PR.

File: draft-ietf-httpbis-resumable-upload.md [hunk 1]
```diff
@@ -522,4 +522,6 @@ However, from experience with the tus project, users are often asking for a way
 
 This document is based on an Internet-Draft specification written by Jiten Mehta, Stefan Matsson, and the authors of this document.
 
+The Initial version of this document was inspired by the [tus v1 protocol](https://tus.io/) which provides a specification and working implementations for a resumable file upload protocol over HTTP. Memebers of its community helped significantly in creating the early Internet-Draft.
```
Comment (LPardue) [inline draft-ietf-httpbis-resumable-upload.md:525]: s/memebers/members

Comment (LPardue) [reply draft-ietf-httpbis-resumable-upload.md:525]: Also s/Initial/initial

Comment (LPardue) [inline draft-ietf-httpbis-resumable-upload.md:525]: Possible tweak if you're after an alternative, up to you if you want to pick it up

```suggestion
The [tus v1 protocol](https://tus.io/) is a specification for a resumable file upload protocol over HTTP. It inspired inspired the early design of this protocol. Members of the tus community helped significantly in the process of bringing this work to the IETF.
```

Comment (Acconut) [reply draft-ietf-httpbis-resumable-upload.md:525]: Shoot, I should not make PRs at such a late hour!

**Comment by Acconut** [comment]
Great suggestions, Lucas! Thank you very much!

**Review by LPardue — APPROVED** [review]

**Comment by LPardue** [comment]
LGTM, thanks for picking it the pen.

**Review by guoye-zhang — APPROVED** [review]

**Pull request merged**